src/main/java/banking/model/Account.java

```diff
@@ -0,0 +1,15 @@
+package banking.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
```

Comment:
Can you use the order that Idea suggests when you use hotkey `Ctrl+N`?

![image](https://user-images.githubusercontent.com/27965136/147115436-f37e4c97-d65d-45d1-a8ae-69d32527eb3f.png)

Reply:
ordered